js/nav.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+(function() {
+
+  var currentNav = function() {
+    var navLinks = document.getElementById('nav-menu').getElementsByTagName('a');
+
+    // console.log(navLinks);
+    // used this to understand what I was doing better
+
+    for(var i = 0; i < navLinks.length; i++) {
+      if(document.location.href.indexOf(navLinks[i].href) >= 0) {
+        return navLinks[i].parentNode.className='nav-link-active';
+      }
```

Comment:
this is good, but I wanted to give you another `if` conditional possibility. If something doesn't exist in an array, it has an index of `-1` so you could write `if(document.location.href.indexOf(navLinks[i].href) !== -1)`